examples/test_fixtures/main.tf

```diff
@@ -0,0 +1,54 @@
+terraform {
+  required_version = "~> 0.10.6"
+}
+
+provider "aws" {
+  region  = "${var.aws_region}"
+  version = "1.0.0"
+}
+
+provider "template" {
+  version = "1.0.0"
```

Comment:
Do you actually need a specific version?

Reply:
Make sure it works with the latest stable version.

Reply:
Again moved to optimistic. Version constraints are important and will only become more important moving forward. I would only feel comfortable unsetting this if the module were actually tested against those previous versions.